approaches/byte-pair-encoding.md

```diff
@@ -5,3 +5,5 @@ layout: coming_soon
 title: Byte-pair encoding
 description: A data compression algorithm for machine translation
 ---
+
+Tokenisation is fundamental to understanding human language. Algorithms for tokenisation can be based on word, subword or character. In this article, we discuss about subword-based tokenisation followed by Byte-pair encoding (BPE) algorithm.
```

Comment:
- The first sentence should describe and define the topic - byte-pair encoding for machine translation - so for example "**Byte-pair encoding** (**BPE**) is a compression algorithm that is used for [tokenisation](/tokenisation) for neural machine translation."
- Don't define tokenisation here, just create the tokenisation article, then link to it
- Write like Wikipedia not an academic paper, so no "We discuss..."